packages/core/src/components/common/Navbar/navbar.module.scss

```diff
@@ -186,7 +186,6 @@
 
   [data-fs-navbar-header] {
     padding: var(--fs-navbar-header-padding);
```

Comment:
```suggestion
    padding: var(--fs-navbar-header-padding);
    overflow-x: clip;
```

Reply:
fixes this error on mobile ;)
![image](https://user-images.githubusercontent.com/3356699/226769794-ecd3273b-93c1-49a9-be3d-e9fcae6379eb.png)